dist/pythonlibs/riotctrl_shell/requirements.txt

```diff
@@ -0,0 +1 @@
+git+ssh://git@github.com/RIOT-OS/riotctrl
```

Comment:
```suggestion
git+ssh://git@github.com/RIOT-OS/riotctrl
pytest
flake8
```

Reply:
Shouldn't this rather be a dependency of testing? I know tox can be configured as such.

Reply:
You are right. I edited the tox.ini above. If it works please ignore the change in requirements.txt :)